### PR TITLE
Fix/#41 links

### LINKED ISSUE #41: Dead link (to ReturnValues) on http://site.mockito.org/
On [_posts/2000-01-15-more.md](https://github.com/mockito/mockito.github.io/blob/master/_posts/2000-01-15-more.md) the link to ReturnValues returns an error (as [ReturnValues](https://static.javadoc.io/org.mockito/mockito-core/1.9.5/org/mockito/ReturnValues.html) was deprecated).

Also the link to Answer points to http://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/Mockito.html#field_summary.
Shouldn't it be http://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/Mockito.html#answer_stubs or http://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/stubbing/Answer.html ?

Thanks for the site!

### PR DESCRIPTION
**Pull request by DamianFekete**
Fixes #41

### CODE REVIEW

File: _posts/2000-01-15-more.md [hunk 1]
```diff
@@ -8,14 +8,14 @@ fa-icon: book
...
-* [`spy()`](http://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/Mockito.html#spy(T))/[`@Spy`](http://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/Spy.html):
+* [`spy()`](http://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/Mockito.html#spy-T-)/[`@Spy`](http://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/Spy.html):
   partial mocking, real methods are invoked but still can be verified and stubbed
 * [`@InjectMocks`](http://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/InjectMocks.html): automatically inject mocks/spies fields annotated with `@Spy` or `@Mock`
-* [`verify()`](http://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/Mockito.html#verify(T)): to check methods were called with given arguments
+* [`verify()`](http://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/Mockito.html#verify-T-): to check methods were called with given arguments
     * can use flexible argument matching, for example any expression via the [`any()`](http://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/Matchers.html#any())
```
Comment (TimvdLippe) [inline _posts/2000-01-15-more.md]: This should be https://static.javadoc.io/org.mockito/mockito-core/2.21.0/org/mockito/ArgumentMatchers.html#any--

**Review by TimvdLippe — CHANGES_REQUESTED** [review]
1 more fix, then LGTM

**Review by TimvdLippe — APPROVED** [review]
Thanks!

**Comment by DamianFekete** [comment]
I haven't seen that Matchers is deprecated and that is why I didn't want to change the link.

**Pull request merged**